scripts/removeSshKeyResult.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# removeSshKeyResult.sh - run by a custom script deployment to
+#   1. uses the Azure CLI to delete a Deployment Script resource
+#   *. use Bicep's replace() function to
+#        inject values wherever resourceGroupNameDefaultValue appears
+#        inject values wherever generateSshKeyScriptNameDefaultValue appears
+
+az deployment-scripts delete \
```

Comment:
validated post deployment execution via Azure CLI command -> az deployment-scripts list; and confirmed script execution and deletion between current Rg with branch fix and previous Rg without branch fix. Approved